src/components/MainView/index.jsx

```diff
@@ -19,24 +18,15 @@ const MainView = ({
 }) => (
   <div key={ldapEmail}>
     <h2 className={classes.header}>{partialOrg[ldapEmail].cn}</h2>
```

Comment:
Once you move the name of the person within the header with the tabs you will be able to get rid of this `h2` and the `div` just below.

Reply:
I'm not sure that I understand this comment fully. I moved the `<h2>` into Maintabs, so it will show up in every tabs. Please help me to explain more if I did wrongly. Thanks.